src/vsr/replica.zig

```diff
@@ -1294,7 +1294,7 @@ pub fn ReplicaType(
                 .trace_emit_timeout = Timeout{
                     .name = "trace_emit_timeout",
                     .id = replica_index,
-                    .after = 100,
+                    .after = 1000,
```

Comment:
This is every 15 seconds, right? Tick is 15 ms?

Reply:
This is 10 seconds, since tick is 10ms:

```
    tick_ms: u63 = 10,
```

Reply:
🤯 I was 110% sure it is 15! One of my recurring thoughts is that "gosh, we should increase our tick from 15 ms to 16 ms, to be _exactly_ like a game". 

I wonder where did I get 15 from?